packages/editor/src/components/reusable-blocks-buttons/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { _BlockSettingsMenuPluginsExtension } from '@wordpress/block-editor';
+import { withSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import ReusableBlockConvertButton from './reusable-block-convert-button';
+import ReusableBlockDeleteButton from './reusable-block-delete-button';
+
+function ReusableBlocksButtons( { clientIds } ) {
+	return (
+		<_BlockSettingsMenuPluginsExtension>
```

Comment:
Aside: Should we consider renaming these to `__unstable` or `__experimental` . Was that the idea with them, or some separate concept of "internal" ? We have other prior art even with components, e.g. `__experimentalAsyncModeProvider`.

Reply:
right, it's I think an internal thing but I agree that consistency is better.